src/AccessibilityInsights.Extensions.Telemetry/TelemetryClientWrapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AccessibilityInsights.Extensions.Telemetry
+{
+    internal class TelemetryClientWrapper : ITelemetryClientWrapper
+    {
+        private readonly TelemetryClient _client;
+
+        internal TelemetryClientWrapper(TelemetryClient client)
+        {
+            if (client == null)
+                throw new ArgumentNullException(nameof(client));
+
+            _client = client;
+        }
+
+        async void ITelemetryClientWrapper.TrackEvent(EventTelemetry data)
+        {
+            await Task.Run(new Action(() => _client.TrackEvent(data))).ConfigureAwait(false);
+        }
+
+        void ITelemetryClientWrapper.TrackException(Exception e, Dictionary<string, string> contextProperties)
+        {
+            _client.TrackException(e, contextProperties);
```

Comment:
This is retaining the synchronous behavior of TrackException. @karanbirsingh, what are your thoughts on making it asynchronous, like the events are?

Reply:
I had assumed both were fire & forget, so I'd be OK with that

Reply:
Made them both fire & forget